client/global.c

```diff
@@ -0,0 +1,11 @@
+#include "global.h"
+
+int g_xdag_state = XDAG_STATE_INIT;
+int g_xdag_testnet = 0;
+int g_xdag_run = 0;
+enum xdag_field_type g_block_header_type = XDAG_FIELD_HEAD;
+struct xdag_stats g_xdag_stats;
+struct xdag_ext_stats g_xdag_extstats;
+int g_disable_mining = 0;
+enum xdag_type g_xdag_type = 0;
+char *g_coinname, *g_progname;
```

Comment:
Emmm, there is one extra character on windows.

Reply:
Sorry, I didn't understand what is wrong here.